### PR TITLE
doc: Resolves a spelling mistake

### PR DESCRIPTION
**Pull request by tarrenj**
Resolves spelling mistake in CONTRIBUTING.md
"behaviour" -> "behavior"

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ There are three categories of refactoring, code only moves, code style fixes,
 code refactoring. In general refactoring pull requests should not mix these
 three kinds of activity in order to make refactoring pull requests easy to
 review and uncontroversial. In all cases, refactoring PRs must not change the
-behaviour of code within the pull request (bugs must be preserved as is).
+behavior of code within the pull request (bugs must be preserved as is).
```
Comment (jeandudey) [inline CONTRIBUTING.md]: Just the difference in accents?

**Comment by promag** [comment]
What's wrong with behaviour?

**Comment by kristapsk** [comment]
NACK, It's not spelling mistake, I prefer British English. :)

**Comment by tarrenj** [comment]
Whoops!  I honestly didn't know there were different spellings for that word.  Closing this PR.

**Pull request closed without merging**